README.rst

```diff
@@ -363,7 +364,7 @@ and password.
     $user = "<user_name>";
     $password = "<password>";
 
-    $dbh = new PDO("snowflake:account=$account", $user, $password);
+    $dbh = new PDO("snowflake:account=$account;warehouse=<wh_name>;database=<db_name>;schema=<schema_name>", $user, $password);
```

Comment:
we are using variables in lines above - could we use it also here?
to have connection string `"snowflake:account=$account;warehouse=$warehouse;database=$database;schema=$schema"`

Reply:
surely we can, added